core-services/prow/02_config/openshift/managed-scripts/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/managed-scripts
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/managed-scripts
+  review_acts_as_lgtm: true
+plugins:
+  openshift/managed-scripts:
+    plugins:
+    - approve
```

Comment:
In order for `lgtm` and `approve` to make sense, you also need to configure Tide, their consumer (see other repositories `_prowconfig.yaml` configuration).

Reply:
@petr-muller Could you check again?